PD configs: fix the log-related configs (#7421)
pingcap/docs: command-line-flags-for-pd-configuration.md

## `--join`

- Join the cluster dynamically
- Default: `""`
- If you want to join an existing cluster, you can use `--join="${advertise-client-urls}"`, the `advertise-client-url` is any existing PD's, multiply advertise client urls are separated by comma.

## `-L`

- The log level
- Default: `"info"`

> - Optional: `"debug"`, `"info"`, `"warn"`, `"error"`, `"fatal"`

Reviewer: ```suggestion
- Optional values: `"debug"`, `"info"`, `"warn"`, `"error"`, `"fatal"`
```

## `--log-file`

- The log file
- Default: `""`

> - If this flag is not set, logs will be written to "stderr".

Reviewer: ```suggestion
- If this option is not set, logs will be written to "stderr". If this option is set, logs are output to the corresponding file.
```

> - If this flag is not set, logs will be written to "stderr".

Author: ```suggestion
- If this flag is not set, logs will be written to "stderr". If this flag is set, logs are output to the corresponding file.
```
